pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -719,7 +741,10 @@ def append_tk_command_to_qiskit(
         ) from error
     params = _get_params(op, symb_map)
     g = gatetype(*params)
-    qcirc.global_phase += phase * sympy.pi
+    if type(phase) == float:
+        qcirc.global_phase += float(phase * sympy.pi)
```

Comment:
May as well use `numpy.pi` or `math.pi` here?

Reply:
done in [use numpy.pi](https://github.com/CQCL/pytket-qiskit/pull/245/commits/143a76e6223d91df42a61a5267e24c631042e9ed)
[143a76e](https://github.com/CQCL/pytket-qiskit/pull/245/commits/143a76e6223d91df42a61a5267e24c631042e9ed)